Submitted By: Joe Ciccone <jciccone@gmail.com
Date: 2010-08-08
Initial Package Version: 2.12
Upstream Status: Unknown
Origin: Joe Ciccone
Description: Seperates the normal and implcit rules, make 3.82 complains
             when they are defined on the same line.

diff -Naur eglibc-2.12.orig/manual/Makefile eglibc-2.12/manual/Makefile
--- eglibc-2.12.orig/manual/Makefile	2010-03-31 19:12:26.000000000 +0000
+++ eglibc-2.12/manual/Makefile	2010-08-08 20:29:22.905807775 +0000
@@ -243,7 +243,11 @@
 .PHONY: stubs
 stubs: $(objpfx)stubs
 endif
-$(objpfx)stubs ../po/manual.pot $(objpfx)stamp%:
+$(objpfx)stubs ../po/manual.pot:
+	$(make-target-directory)
+	touch $@
+
+$(objpfx)stamp%:
 	$(make-target-directory)
 	touch $@
 
